plugins/AD7303.c

```diff
@@ -54,18 +54,18 @@ static GtkWidget *radio_waveform;
 static GtkWidget *databox;
 static GtkWidget *preview_graph;
 
-static GdkColor color_background = {
+static GdkRGBA color_background = {
```

Comment:
Should you also add the alpha channel? (besides red, green and blue)

Reply:
added alpha channel for all structures